src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -66,27 +59,146 @@ The 'Show' instance produces symmetrically closed expressions:
 
 @show (1     :: SymmetricRelation Int) == "vertex 1"
 show (1 * 2 :: SymmetricRelation Int) == "edges [(1,2),(2,1)]"@
+
+The 'Graph' instance respects the comutativity of connect.
```

Comment:
You have already mentioned this above.

Reply:
Same for `Num` -- no need to repeat this.

Reply:
Got it!